docs: add kb manually update keyman for windows (#2206)
keymanapp/help.keyman.com: knowledge-base/kb0119.md

> # Keyman for Windows stays on same version after upgrade

Reviewer: This reads is a little clearer to me

```suggestion
# Keyman for Windows version stays the same after upgrade
```

> # Keyman for Windows version stays the same after upgrade

Reviewer 2: ```suggestion
# BUG: Keyman for Windows version 18.0.235 or 18.0.236 do not automatically upgrade to newer versions
```

## Summary

> In some cases, when upgrading Keyman for Windows 18 to a newer point release, the software may remain on the same version after the update process.

Reviewer 2: ```suggestion
In some cases, when upgrading Keyman for Windows 18.0.235 or 18.0.236 to a newer point release, the software may remain on the same version after the update process.
```

This happens because the installer for the currently installed version is still cached on the local machine. When the update process starts, the cached installer is used instead of the newly downloaded version, so the upgrade does not complete as expected.

## Manual update to the latest version

> To resolve this issue, install the latest version manually. You do not need to uninstall the current version before installing the new version.

Reviewer 2: ```suggestion
To resolve this issue, install the latest version (18.0.237 or later) manually. You do not need to uninstall the current version before installing the new version.
```
